OpenTap.Plugins.PNAX/Instrument/PNAModulationDistortion.cs

```diff
@@ -197,7 +197,19 @@ public enum MODTableSetupEVMEnum
         EVMDistUn21 = 8,
         [Display("EVM Norm")]
         [Scpi("EVM Norm")]
-        EVMNorm = 16
+        EVMNorm = 16,
```

Comment:
Does this change to an enum constitute a "breaking change"? At the very least this should be a "minor version" bump, not just a patch release. What are your thoughts @c-montes / @alexwang-keysight / @bdirenzo-keysight / @alnlarsen / @rmadsen-ks ?

Reply:
I added additional options for this enum based on a request from the IST, this was part of the Modulation Distortion implementation I completed last week. Nobody is using this besides the IST for now.